src/js/components/List/List.js

```diff
@@ -232,10 +229,8 @@ const List = React.forwardRef(
             (onClickItem || onOrder) && data && data.length
               ? () => {
                   const min = onOrder ? 1 : 0;
-                  const max = onOrder ? (data.length * 2) - 2 : data.length - 1;
-                  setActive(
-                    active >= min ? Math.min(active + 1, max) : min,
-                  );
+                  const max = onOrder ? data.length * 2 - 2 : data.length - 1;
```

Comment:
seems like removal of parentheses for calculations is consistent across files, but just want to confirm it is expected.

Reply:
Confirmed. I believe this is done by the `eslint-plugin-prettier`.